packages/google_maps_flutter/google_maps_flutter_web/lib/src/map_styler.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:js_interop';
+
+/// The interop type for a Google Maps Map Styler.
+///
+/// See: https://developers.google.com/maps/documentation/javascript/style-reference#stylers
```

Comment:
Due to the removal of the `dart:js_util` import, I added this type to be able to remove the `jsify()` call. See the documentation in the link above.